### PR TITLE
This week in os 14

### PR DESCRIPTION
**Pull request by emmasofiah2o**
- Created file
- Added og:image
- Added Ember PR to config file
- Check unknown links

### CODE REVIEW

File: _posts/2022-08-15-this-week-in-os-14.md [hunk 1]
```diff
@@ -0,0 +1,194 @@
...
+
+## Unknown
+
+- [calibreapp/image-actions]
+  [#126](https://github.com/calibreapp/image-actions/pull/126) fix(entrypoint):
+  exit(0) when pull_request.<event> is not 'synchronize' or 'opened'
+  ([@oscard0m])
+- [locks/netrunner] [#2](https://github.com/locks/netrunner/pull/2) Create
+  ci.yml ([@locks])
+- [mansona/lint-to-the-future-dashboard-action]
```
Comment (mansona) [inline _posts/2022-08-15-this-week-in-os-14.md]: this should be in lint-to-the-future 😉

File: _posts/2022-08-15-this-week-in-os-14.md [hunk 2]
```diff
@@ -0,0 +1,194 @@
...
+  just use modules ([@mansona])
+- [mansona/showdown-section-groups]
+  [#3](https://github.com/mansona/showdown-section-groups/pull/3) Fix the repo
+  metadata and add a more useful readme ([@mansona])
+- [mansona/showdown-section-groups]
+  [#2](https://github.com/mansona/showdown-section-groups/pull/2) breaking: drop
+  support for Node < 14 (EOL) ([@mansona])
+- [mansona/showdown-section-groups]
+  [#1](https://github.com/mansona/showdown-section-groups/pull/1) move to GitHub
+  actions ([@mansona])
```
Comment (mansona) [inline _posts/2022-08-15-this-week-in-os-14.md]: I would just put showdown-section-groups in Javascript 👍

File: _posts/2022-08-15-this-week-in-os-14.md (hunk 1 above)
Comment (emmasofiah2o) [reply _posts/2022-08-15-this-week-in-os-14.md]: Ah true!

File: _posts/2022-08-15-this-week-in-os-14.md (hunk 2 above)
Comment (emmasofiah2o) [reply _posts/2022-08-15-this-week-in-os-14.md]: I don't think I can so sub-paragraphs actually, let me try

File: _posts/2022-08-15-this-week-in-os-14.md [hunk 3]
```diff
@@ -0,0 +1,197 @@
...
+- [locks/netrunner] [#2](https://github.com/locks/netrunner/pull/2) Create
+  ci.yml ([@locks])
+- [mtgred/netrunner] [#6505](https://github.com/mtgred/netrunner/pull/6505) Fix
+  deckbuilder's "Import deck" ([@locks])
+- [mtgred/netrunner] [#6504](https://github.com/mtgred/netrunner/pull/6504)
+  Implement CI as a GitHub Action ([@locks])
+- [mtgred/netrunner] [#6503](https://github.com/mtgred/netrunner/pull/6503) Fix
+  "Swap sides" chat messages in pending lobby ([@locks])
+- [mtgred/netrunner] [#6491](https://github.com/mtgred/netrunner/pull/6491)
+  "Into The Depths" install from the stack, not R&D ([@locks])
```
Comment (locks) [inline _posts/2022-08-15-this-week-in-os-14.md]: These can be removed, they're not part of 20%.

File: _posts/2022-08-15-this-week-in-os-14.md [hunk 4]
```diff
@@ -0,0 +1,197 @@
...
+- [octokit/auth-oauth-user.js]
+  [#71](https://github.com/octokit/auth-oauth-user.js/pull/71) ci(test): use
+  Octokit's 'test' Workflow ([@oscard0m])
+- [octokit/webhooks-methods.js]
+  [#66](https://github.com/octokit/webhooks-methods.js/pull/66) ci(test): re-use
+  Octokit's test.yml action ([@oscard0m])
+
+## Unknown
+
+- [calibreapp/image-actions]
```
Comment (oscard0m) [inline _posts/2022-08-15-this-week-in-os-14.md]: This is a GitHub Action to compress images in a Pull Request. Is uses JavaScript so I guess we can put it there.

**Review by BobrImperator — APPROVED** [review]

**Review by oscard0m — APPROVED** [review]

**Pull request merged**